### PR TITLE
Adds a new module for Admin modsuits. (+ Chrono, Debug)

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
image

## Why It's Good For The Game
As much as Chronos are a reference to a time when even SSBlackbox could be nuked IC, they're still fun adminbus tools. This brings the joke full circle.

## Changelog

:cl:
expansion: A new Kinesis+ module has rolled out to Chrono Legionaries. Consequently, there's now support for MOD kinesis modules that can handle living mobs, and even variable stun times for it.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
neat

File: code/modules/mod/modules/module_kinesis.dm [hunk 1]
```diff
@@ -248,3 +253,13 @@
 	given_turf = locate(mob_x+our_x-round(view[1]/2),mob_y+our_y-round(view[2]/2),mob_z)
 	given_x = round(icon_x - world.icon_size * our_x)
 	given_y = round(icon_y - world.icon_size * our_y)
+
+/obj/item/mod/module/anomaly_locked/kinesis/chrono
+	name = "MOD kinesis+ module" //Chrono Legionarrie variant. You know why, deep in your heart.
```
Comment (Fikou) [inline code/modules/mod/modules/module_kinesis.dm:258]: legionnaire
would be better if you called it advanced or kinesis plus

File: code/modules/mod/modules/module_kinesis.dm [hunk 2]
```diff
@@ -248,3 +253,13 @@
...
+	desc = "A modular plug-in to the forearm, this module was recently redeveloped in secret. \
+	The bane of all ne'er-do-wells, the kinesis+ module is a powerful tool that allows the user to \
+	manipulate the world around them. Like it's older counterpart, it's capable of manipulating structures, \
+	machinery, vehicles, and, thanks to the fruitful efforts of it's creators - living beings. They can, however, \
+	still struggle after an initial burst of inertia."
```
Comment (Fikou) [inline code/modules/mod/modules/module_kinesis.dm:263]: tab these

File: code/modules/mod/modules/module_kinesis.dm [hunk 3]
```diff
@@ -18,6 +18,8 @@
 	var/grab_range = 5
 	var/hit_cooldown_time = 1 SECONDS
 	var/movement_animation
+	var/mobs_allowed = FALSE
+	var/living_stun_length = 100
```
Comment (Fikou) [inline code/modules/mod/modules/module_kinesis.dm:22]: use second defines

Comment (Fikou) [reply code/modules/mod/modules/module_kinesis.dm:22]: also doc it

**Comment by Fikou** [comment]
admin suits should have it

File: code/modules/mod/modules/module_kinesis.dm (hunk 1 above)
Comment (unit0016) [reply code/modules/mod/modules/module_kinesis.dm:258]: 💀 
I mean fair, I just don't want a mapper accidentally putting it somewhere it shouldn't be. Oh well, that's a bridge I'll cross another day.

**Comment by GolfinhoVoador** [comment]
add the admin boxing gloves or something like that as a module for it

**Comment by Spc-Dragonfruits** [comment]
Assistants aren't supposed to have guns.

**Pull request closed without merging**